### ISSUE TITLE
Horrible audio quality after transcoding

### ISSUE THREAD

**Issue by kerenon** [Chocobozzz/PeerTube#1837]
I'm not really sure this is a bug report... a question maybe. I've taken an old Beat Saber video. Uploaded it to youtube, to my peertube instance, and the original mp4 file to google drive

Youtube: https://www.youtube.com/watch?v=StIH2S2KfpE
Peertube: https://peertube.kerenon.com/videos/watch/dbca023b-a521-448c-97d8-98e7650730db
Original: https://drive.google.com/file/d/1RdMHZFNbDZRTH-JnQZ8OWoRaDsFQyhMM/view?usp=sharing

I don't think I need to explain the problem in detail. The Peertube version is just horrible. The 144p(!) youtube version has better sound quality than the peertube 720p. Is this a bitrate problem? Is this intentional? Is it because of the ffmpeg version I'm using (see bottom). Can I just fork the code and bump up the bitrate, or are there some other more indepth modifications required? I've found an issue from last year regarding audio quality, but it seems it was resolved. Is this a regression?

I really, really like peertube, and want to make this work... somehow :)

**Additional information**
* PeerTube version or URL: 1.3.0-rc.2
* Browser name/version: any browser
* NodeJS version: v8.16.0
* ffmpeg version:  4.1.3-static https://johnvansickle.com/ffmpeg/; built with gcc 6.3.0 (Debian 6.3.0-18+deb9u1) 20170516

**Comment by Chocobozzz** [comment]
I'm sorry but I can't reproduce: the sound doesn't sound horrible to me on peertube. Maybe it's a playback bug? Could you try to download the peertube video file, and play it with VLC for example?

**Comment by kerenon** [comment]
Doesn't matter where I play the video. Downloading and examining it shows that it has a 128 kbps AAC audio stream, and in my books that's horrible. Makes listening to audio really unpleasant. I've managed to fix it for myself, by adding `command = command.outputOption('-b:a 384k')` to the transcode function in ffmpeg-utils.ts, so if the sound quality is this bad "by desing", I just have to maintain a fork for my small modifications.

**Comment by Chocobozzz** [comment]
> Downloading and examining it shows that it has a 128 kbps AAC audio stream

But it seems the original file you uploaded on google drive has a 128 kb/s aac audio stream:

```
$ ffprobe Beat\ Saber_20181208213902.mp4
...
 Stream #0:1(eng): Audio: aac (LC) (mp4a / 0x6134706D), 48000 Hz, stereo, fltp, 128 kb/s (default)
```

**Comment by kerenon** [comment]
Then it seems I uploaded the wrong file :D

**Comment by kerenon** [comment]
I'm taking the sample video files and my instance offline. Can't really use it like this for anything I've planned, and I need my server for something else right now.
Node / Js is so very alien (i.e.: hate it with all my hearth :D) to me, and without really knowing what I'm doing, tinkering with the code fixes my problem with the audio but it also creates some new ones. I was curious about the new audio upload feature, so I've tried uploading a lossless flac track, so I can rule out the double encoding problem, and for the result... I can only say: **no**. It is just bad. I don't know if it's the encoder or if it's just bitrate-starwed but it is bad, and literally everyone I've shown the results notices the bad quality. I think I'll just sit back and wait, maybe someday we do get our bitrate-slider/selector/something :)

As this is not really a bug (I've noticed for some people, this quality is enough), I think we can close this
